src/components/organisms/CourseList/CourseList.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { IProps } from './CourseList.model';
+import { Container, CourseWrapper, Line } from './CourseList.styled';
+import CourseElement from './CourseElement';
+
+const CourseList = ({ courses }: IProps): JSX.Element => {
+  const lastIndex = courses.length - 1;
+  return (
+    <div>
+      {courses.map((course, index) => (
+        <Container>
+          <CourseWrapper key={course.id}>
+            <CourseElement {...course} tags={course.tags} />
+          </CourseWrapper>
+          {index !== lastIndex && <Line />}
```

Comment:
@Krysik is it possible to do this in css using ::after and ::last-child? Could you check it?